src/wazuh_db/wdb_global.c

```diff
@@ -1158,6 +1158,10 @@ cJSON* wdb_global_get_groups_integrity(wdb_t* wdb, os_sha1 hash) {
 int wdb_global_get_agent_max_group_priority(wdb_t *wdb, int id) {
     sqlite3_stmt *stmt = wdb_init_stmt_in_cache(wdb, WDB_STMT_GLOBAL_GROUP_PRIORITY_GET);
 
+    if (stmt == NULL) {
```

Comment:
I think we should remove this extra white space.

Reply:
Removed